src/utils/styleConstants.ts

```diff
@@ -21,7 +22,7 @@ export const COLORS = {
   LINK: "#93C5FD",
   NEUTRAL_WHITE: "#FFF",
   NEUTRAL_BLACK: "#000",
-  ERROR: "#FCA5A5",
```

Comment:
The color of the error is this shade, however, maybe the people who made the colors did not update the colors or did not follow the colors established by the firm, it would be good if you come to an agreement with the team that made the colors so that there is no discrepancy.

![image](https://github.com/user-attachments/assets/4448179a-a806-4ea7-8bac-c463bc7cfa05)

Reply:
If you check the figma, this color is not used anymore, instead of it, they used just red color (#f00) in the components. The idea of the implementation of a view is that it has to be the most similar to the design, with this color it does not happen.

Reply:
In this case it would not be your change but the change of the group that made the screens, it would be good if you talked to the group so that it is not strange at the moment that another person tests it.

Reply:
I've already talked with Diego, the error color variable was changed to red #f00.